src/proxy-animation.js

```diff
@@ -1596,7 +1598,8 @@ function parseOneAnimationDelay(delay, defaultOffset) {
 
   const parts = delay.split(' ');
 
-  if(parts.length == 2 && !parts[1].endsWith('%'))
+  if(!ANIMATION_DELAY_NAMES.includes(parts[0]) ||
+    (parts.length == 2 && !parts[1].endsWith('%')))
```

Comment:
Thanks for adding the additional validation.  We should also verify that the parseFloat call succeeds (didn't return NaN). Calling with 'cover rubbish%" will now fail with:

Uncaught TypeError: Failed to execute 'percent' on 'CSS': The provided double value is non-finite.

Reply:
Done